README.md

```diff
@@ -194,6 +200,90 @@ Detailed logs and results are available on Neoload Web. To get the url of the cu
 neoload logs-url                        # The URL to the test in Neoload Web
 ```
 
+### The test-results vs. report subcommands
+
+The 'test-results' subcommand is intended for direct operational queries against high-level API data.
+
+The 'report' subcommand is intended to simplify not only common data exporting needs, but also provide
+ templating capabilities over a standard, correlated data model. In contrast to the test-results
+ subcommand, 'report' can be used to generate as well as transform test result data.
+
+### Exporting Transaction CSV data
+```
+Usage: neoload report [OPTIONS] [[single|trends]] [NAME]
+neoload report --template builtin:transactions-csv single cur > temp.csv
+```
+
+### Filtering export data by timespan
+In many load tests, ramp-up and spin-down time is considered irrelevant to calculate into aggregate statistics,
+ such as how when warming up, systems may produce higher-than-expected latencies until a steady state is reached.
+
+Therefore, the NeoLoad CLI allows for export of particular time ranges by providing a timespan filter.
+
+```
+neoload report --template builtin:transactions-csv --filter "timespan=5m-95%" single cur
+neoload report --template builtin:transactions-csv --filter "timespan=15%" single cur
+neoload report --template builtin:transactions-csv --filter "timespan=-90%" single cur
```

Comment:
Idem remove "single" here and below commands

Reply:
0ac6fe6 Updated README to remove unnecessary options when defaults are sufficient (single cur)